File: 10 - Python/02 - Estrutura Condicional/Exercício 09.py
#Exercício - Lanchonete

codigo = int(input("Código do produto comprado: "))
quantidade = int(input("Quantidade comprada: "))

if codigo == 1:
    valor = 5.00 * quantidade
elif codigo == 2:
    valor = 3.50 * quantidade
elif codigo == 3:
    valor = 4.80 * quantidade
elif codigo == 4:
    valor = 8.90 * quantidade
elif codigo == 5:
    valor = 7.32 * quantidade
else:
    print("Código inválido!")

print("Valor a pagar: R$", valor)

File: 06 - C/Exercícios/02 - Estrutura Condicional/Exercício 09.c
//Exercício - Lanchonete

#include <stdio.h>

int main() {

    int codigo, quant;
    double valor;

    printf("Codigo do produto comprado: ");
    scanf("%d", &codigo);

    printf("Quantidade comprada: ");
    scanf("%d", &quant);

    switch (codigo){
        case 1:
            valor = 5.0 * quant;
            break;
        case 2:
            valor = 3.5 * quant;
            break;
        case 3:
            valor = 4.8 * quant;
            break;
        case 4:
            valor = 8.9 * quant;
            break;
        case 5:
            valor = 7.32 * quant;
            break;
    }

    printf("Valor a pagar: R$%.2lf", valor);

    return 0;

}